.buildkite/pull-requests.json

```diff
@@ -9,7 +9,7 @@
       "commit_status_context": "ml-cpp-ci",
       "build_on_commit": true,
       "build_on_comment": true,
-      "trigger_comment_regex": "^(?:(?:buildkite +)(?<action>build|debug|run_qa_tests|run_pytorch_tests) *(?: *on *(?<platform>(?:[ ,]*(?:windows|linux|mac(os)?))+))?) *(?<arch>(?:[ ,]*aarch64|x86_64)+)?$",
+      "trigger_comment_regex": "^(?:(?:buildkite +)(?<action>build|debug|run_qa_tests|run_pytorch_tests) *(?: for ES_BRANCH=(?<branch>([.0-9a-zA-Z]+)))? *(?:with STACK_VERSION=(?<version>([.0-9]+)))? *(?: *on *(?<platform>(?:[ ,]*(?:windows|linux|mac(os)?))+))?) *(?<arch>(?:[, ]*aarch64|x86_64)+)?$",
```

Comment:
Thanks for working on this. 

Can I use this PR to add another optional match for QAF_TESTS_TO_RUN as well? :)
In this way, it can choose the group of test to run. for example, if it is an inference change, we don't have to run Anomaly Detection tests.

QAF_TESTS_TO_RUN should be handled separately from ES_BRANCH and STACK_VERSION, and totally optional. If it is not defined, all tests will be run.  maybe another `*(?: QAF_TESTS_TO_RUN=...`?

Reply:
Ha! Sure, why not? It's a good suggestion, thanks Wei.